WalletWasabi.Fluent/ViewModels/MainViewModel.cs

```diff
@@ -183,13 +183,6 @@ private void RegisterViewModels()
 				return _settingsPage;
 			});
 
-		PrivacySettingsTabViewModel.RegisterLazy(
-			() =>
-			{
-				_settingsPage.SelectedTab = 1;
-				return _settingsPage;
-			});
-
```

Comment:
Indexes are broken now.

`NetworkSettingsTabViewModel.RegisterLazy` -> _settingsPage.SelectedTab = 1;
`BitcoinTabSettingsViewModel.RegisterLazy` -> _settingsPage.SelectedTab = 2;

On the other hand, https://github.com/zkSNACKs/WalletWasabi/pull/7070 will add a setting to the Privacy tab, might be easier to not remove and just leave it empty for now?

Reply:
Added back empty.